Packs/ServiceNow/Integrations/ServiceNowv2/Classifiers/classifier-ServiceNow_Classifier.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
note that almost all files are located in the wrong place
they should be in the pack in `Packs/ServiceNow`, not under `Integrations/ServiceNowv2`